docs/reference/contributing/guidelines/workflow.md

```diff
@@ -90,6 +90,8 @@ Release: feature
 
 Updating target implementation (adding a new target or updating already supported target) is a change for a patch release.
 
+The test report must be part of the pull request for a new target addition. It must pass all Mbed OS functional and system validation tests for the current Mbed OS major release including all Mbed OS supported toolchains.
```

Comment:
you might also want to specify a report for which tests as this might not be 100% clear to a new platform developer.

Reply:
I would expect to run `mbed test` for the target (all testing available done). Will check